### ISSUE TITLE
PTN000623091 not all cdk

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#3794]
* **PTHR ID & PTN node:**

GO:0004693 | cyclin-dependent protein serine/threonine kinase activity | IBA&nbsp;with&nbsp;Q8IJQ1&nbsp;,&nbsp;O94921&nbsp;,&nbsp;P50750&nbsp;,&nbsp;FBgn0004106&nbsp;,&nbsp;DDB_G0272813&nbsp;,&nbsp;S000006365&nbsp;,&nbsp;CAL0000191263&nbsp;,&nbsp;FBgn0013762&nbsp;,&nbsp;srb10&nbsp;,&nbsp;S000005952&nbsp;,&nbsp;ZDB-GENE-010131-2&nbsp;,&nbsp;P24941&nbsp;,&nbsp;Q00646&nbsp;,&nbsp;lsk1&nbsp;,&nbsp;Q00534&nbsp;,&nbsp;MGI:88357&nbsp;,&nbsp;MGI:88351&nbsp;,&nbsp;cdk9&nbsp;,&nbsp;70486&nbsp;,&nbsp;FBgn0005640&nbsp;,&nbsp;WBGene00000405&nbsp;,&nbsp;C9K505&nbsp;,&nbsp;P61075&nbsp;,&nbsp;P11802&nbsp;,&nbsp;cdc2&nbsp;,&nbsp;70514&nbsp;,&nbsp;621120&nbsp;,&nbsp;DDB_G0288677&nbsp;,&nbsp;S000005963&nbsp;,&nbsp;A0A1D8PDA6&nbsp;,&nbsp;S000001622&nbsp;,&nbsp;S000000364&nbsp;,&nbsp;mcs6&nbsp;,&nbsp;ZDB-GENE-081022-110&nbsp;,&nbsp;FBgn0263237&nbsp;,&nbsp;2319&nbsp;,&nbsp;PTN000623091&nbsp;,&nbsp;621124&nbsp;,&nbsp;P06493&nbsp;,&nbsp;FBgn0019949&nbsp;,&nbsp;pef1&nbsp;,&nbsp;MGI:104772&nbsp;,&nbsp;locus:201176

* **Sequences with problematic annotation (ID + gene/protein name):**

csk1  P36615

* **Type of Issue: Erroneous source or erroneous propagation, or other issue**

erroneous propagation.
csk1 is a "cyclin-dependent kinase activating kinase "
but isn't cyclin dependent itself.

* Add the label 'high priority' if needed. Generally high priority issues affect a lot of proteins, and annotations are incorrect (as opposed to just imprecise).

**Comment by marcfeuermann** [comment]
This is a huge family. I've fixed this where I could but cannot rule out some over problems.

**Comment by sjm41** [comment]
Sorry Marc, but I found another couple of proteins in the CELL DIVISION PROTEIN KINASE (PTHR24056) family that are wrongly getting the "cyclin-dependent protein serine/threonine kinase activity" annotation:

- fly Pk34A (FBgn0028410/Q9VK37) - has 'Glycogen synthase kinase 3, catalytic domain', no evidence/domains supporting CDK activity
- human PRP4B (Q13523) - known to phosphorylate splicing factors, no evidence for cyclin-dependence

Not sure if you review the tree to exclude (or block) annotations to these two, or if it's better just to annotate the family with the parent term 'protein serine/threonine kinase activity'...

**Comment by marcfeuermann** [comment]
No reason to be sorry, it is great that we have input to improve the annotation of those huge families. The root of the tree is localized on a duplication, which is not ideal to make an evolutionary model. I decided to take more time and to change the evolutionary model instead of accumulating small modifications all over the family. 
We have annotated CDK activity to the root and blocked the propagation where it was lost. However, we have more and more branches where we need to block it. So I removed this annotation from the root and put it only on nodes where we are sure to have CDK activity. The root has now "protein ser/thr kinase activity" (which I will also have to block in some places, but there are many less cases). 
Another advantage is that we will have less "NOT" annotations which often cause problems when we update the family trees with newer PANTHER versions. Hope this will be okay... let's follow the future tickets about this family (or not !)
